### PR TITLE
Updated_server

### PR DESCRIPTION
**Pull request by Giladr11**

### CODE REVIEW

File: Scapy_UDP_Server_Client/server.py [hunk 1]
```diff
@@ -6,12 +6,18 @@
...
+        if "loopback" in iface.lower():
+            loopback_iface = iface
+    
+    return loopback_iface
 
-for iface in interfaces:
-    if "loopback" in iface.lower():
-        LOOP_BACK_IFACE = iface
+
+LOOPBACK_IFACE = get_iface()
```
Comment (pazzc) [inline Scapy_UDP_Server_Client/server.py:20]: Assignment of dynamic value into a CONSTANT?

Comment (Giladr11) [reply Scapy_UDP_Server_Client/server.py:20]: yea right i`ll change that

File: Scapy_UDP_Server_Client/server.py [hunk 2]
```diff
@@ -6,20 +6,26 @@
...
+        if "loopback" in iface.lower():
+            loopback_iface = iface
+    
+    return loopback_iface
 
-for iface in interfaces:
-    if "loopback" in iface.lower():
-        LOOP_BACK_IFACE = iface
+
+loopback_iface = get_iface()
```
Comment (pazzc) [inline Scapy_UDP_Server_Client/server.py]: Better. Now, get rid of this line

**Review by pazzc — APPROVED** [review]
great. let's merge

**Pull request merged**